doc/tutorials/contributing-clion.md

```diff
@@ -79,10 +89,11 @@ There you can edit your CMake profiles. To get rid of the root requirement we'll
 add the following CMake options to our "Debug" profile:
 
 ```sh
+-DKDB_DB_HOME="~/.config/kdb/[xyz]/home"
 -DKDB_DB_SYSTEM="~/.config/kdb/[xyz]/system"
 -DKDB_DB_SPEC="~/.config/kdb/[xyz]/spec"
--DKDB_DB_USER=".config/kdb/[xyz]/user"
--DCMAKE_INSTALL_PREFIX=install
+-DKDB_DB_USER="~/.config/kdb/[xyz]/user"
```

Comment:
Unless something changed, this is actually wrong. The `KDB_DB_USER`  path is relative to `KDB_DB_HOME`, so it shouldn't contain `~`. In fact, if you set `KDB_DB_HOME`, you probably don't need to change `KDB_DB_USER` at all.